src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -292,21 +292,6 @@ function PolicyAccountingPage({policy, connectionSyncProgress, isConnectionDataF
         accountingIntegrations,
     ]);
 
-    const headerThreeDotsMenuItems: ThreeDotsMenuProps['menuItems'] = [
-        {
-            icon: Expensicons.Key,
-            shouldShowRightIcon: true,
-            iconRight: Expensicons.NewWindow,
-            text: translate('workspace.accounting.enterCredentials'),
-            onSelected: () => {},
-        },
-        {
-            icon: Expensicons.Trashcan,
-            text: translate('workspace.accounting.disconnect'),
-            onSelected: () => setIsDisconnectModalOpen(true),
-        },
-    ];
-
```

Comment:
@trjExpensify, the three dot menu should only be hidden when there is no accounting connection yet, right?

When there is an accounting connection, we want to keep displaying the three dot menu?

Reply:
this is coming from [this](https://github.com/Expensify/App/issues/41795#issuecomment-2122294227)

Reply:
Hm, no.. I don't see why there would be a three dot overflow menu at the top of the `Accounting` page at all, I think it's a mistake. It has nothing inside that menu. There's an overflow menu inside the actual card on the page that pertains to that integration: 

<img width="1511" alt="image" src="https://github.com/Expensify/App/assets/16232057/81757e6c-f2cb-4747-81b6-ec906cba264d">

Reply:
Why do you think it should be there? What does it do?

Reply:
ah I was just confirming. then we can remove the menu 😄

Reply:
Ah cool, cool!